content/cucumber/api.md

```diff
@@ -772,6 +772,53 @@ import org.junit.runner.RunWith;
 public class RunCukesTest {
 }
 ```
+For example if you want to check whether all feature file steps has corresponding step definitions, you can specify it like this: 
+
+```java
+package mypackage;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(dryRun=true)
```

Comment:
Please use a consistent formatting for the specified option for each of the three examples, either with or without spaces around the `=` sign.